src/pages/FAQ.js

```diff
@@ -65,6 +65,25 @@ export default class FAQ extends Component {
               Once a sufficient percentage of DID holders have approved a pull
               request, DID are automatically issued on the blockchain.
             </p>
+            <p style={{ fontSize: '1.33em', marginTop: '2.2em' }}>
+              <Header as="h3">What id DID?</Header>
+            </p>
+            <p style={{ fontSize: '1.10em' }}>
+              1.)DID are our economic and {'\n'}governance token
```

Comment:
Let's also remove the numbers and use a _bulleted_ Semantic-UI [list](https://semantic-ui.com/elements/list.html#bulleted)